core/src/integrationTest/kotlin/ch/ergon/dope/clauses/SelectQueryIntegrationTest.kt

```diff
@@ -0,0 +1,217 @@
+import ch.ergon.dope.resolvable.expression.unaliased.type.logical.and
+import ch.ergon.dope.resolvable.expression.unaliased.type.relational.isEqualTo
+import ch.ergon.dope.resolvable.expression.unaliased.type.relational.isGreaterThan
+import ch.ergon.dope.resolvable.expression.unaliased.type.relational.isLessThan
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class SelectQueryIntegrationTest : BaseIntegrationTest() {
+    @Test
+    fun `select everything from the test bucket`() {
+        val dopeQuery = QueryBuilder()
+            .selectFrom(
+                testBucket,
+            ).build()
+
+        val queryResult = queryWithoutParameters(dopeQuery)
+
+        assertEquals(15, queryResult.rows.size)
+    }
+
+    @Test
+    fun `select all employees from the test bucket`() {
+        val dopeQuery = QueryBuilder()
+            .selectFrom(
+                testBucket,
+            )
+            .where(
+                typeField.isEqualTo("employee"),
+            ).orderBy(
+                nameField,
+            ).build()
+
+        val queryResult = queryWithoutParameters(dopeQuery)
+
+        assertEquals(5, queryResult.rows.size)
+    }
+
+    @Test
+    fun `select all active clients from the test bucket`() {
+        val dopeQuery = QueryBuilder()
+            .selectFrom(
+                testBucket,
+            )
+            .where(
+                typeField.isEqualTo("client").and(isActiveField.isEqualTo(true)),
+            ).build()
+
+        val queryResult = queryWithoutParameters(dopeQuery)
+
+        assertEquals(2, queryResult.rows.size)
+    }
+
+    @Test
+    fun `select id, type field from the test bucket`() {
+        val dopeQuery = QueryBuilder()
+            .select(
+                idField,
+                typeField,
+            )
+            .from(
+                testBucket,
+            ).orderBy(
+                typeField,
+            ).limit(
+                1,
+            ).build()
+
+        val queryResult = queryWithoutParameters(dopeQuery)
+        val result = queryResult.toMapValues()
+
+        assertEquals(1, result["id"])
+        assertEquals("client", result["type"])
+    }
+
+    @Test
+    fun `select aliased field from the test bucket`() {
+        val alias = "Identification"
+        val dopeQuery = QueryBuilder()
+            .select(
+                idField.alias(alias),
+            )
+            .from(
+                testBucket,
+            ).limit(
+                1,
+            ).build()
+
+        val queryResult = queryWithoutParameters(dopeQuery)
+        val result = queryResult.toMapValues()
+
+        assertEquals(1, result[alias])
+    }
+
+    @Test
+    fun `select where nothing applies from the test bucket`() {
+        val dopeQuery = QueryBuilder()
+            .selectFrom(
+                testBucket,
+            )
+            .where(
+                1.isEqualTo(2),
```

Comment:
Why not just `false`?

Reply:
I want to use as many different functions/constructs as possible in different integration tests; but tbh I already used `.isEqualTo(...)` in other tests, so I can either use another construct that is false or just a "false" in itself.
I would do the first option, what's your opinion?

Reply:
yeah if you already use the equals expression then maybe use something different that also evaluates to false